sdk/python/tests/compiler/testdata/parallelfor_item_argument_resolving.yaml

```diff
@@ -312,5 +312,13 @@ spec:
       taskRef:
         apiVersion: custom.tekton.dev/v1alpha1
         kind: PipelineLoop
-        name: for-loop-for-loop-00000003-3
+        name: for-loop-3
```

Comment:
The generated pipeline-loop crd is still called `for-loop-for-loop-00000003-3`, so the name is not matching over here

Reply:
I generated the YAML files using `make unit_test GENERATE_GOLDEN_YAML=True`

If I change the names in the YAML manually, the unittests fail.
`

Reply:
I think the compiler test has to be updated, the loops crd get updated when i run `TektonCompiler().compile(pipeline, __file__.replace('.py', '.yaml'))`